documentation/modules/changing-precopy-intervals.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+[id="changing-precopy-intervals_{context}"]
+= Changing precopy intervals for warm migration
+
+The snapshots for the precopy stage of warm migration are created at 60-minute intervals by default.
+
+You can change the snapshot interval by patching the `forklift-controller` custom resource (CR).
+
+.Procedure
+
+* Patch the `forklift-controller` CR:
++
+[source,terminal]
+----
+$ oc patch forkliftcontroller forklift-controller -p '{"spec": {"controller_precopy_interval": <6>}}' --type=merge <1>
+----
```

Comment:
ForkliftController is a namespace-scoped resource, so:
oc patch forkliftcontroller/forklift-controller -n openshift-mtv -p '{"spec": {"controller_precopy_interval": <60>}}' --type=merge

Reply:
Done